assets/src/stories-editor/components/rotatable-box/index.js

```diff
@@ -107,6 +108,28 @@ class RotatableBox extends Component {
 		);
 	}
 
+	/**
+	 * Given a rotation angle, finds the closest angle to snap to.
+	 *
+	 * Inspired by the implementation in re-resizable.
+	 *
+	 * @see https://github.com/bokuweb/re-resizable
+	 *
+	 * @param {number} angle
+	 * @return {number} New angle.
+	 */
+	findClosestSnap( angle ) {
```

Comment:
@miina should we perhaps use `memize` here to cache results?

Reply:
I'm not certain. Not sure if we'd need to use the same angle frequently enough / if this calculation is heavy enough for it being worthwhile. WDYT? Or could we move this function out of this component to helpers so that all the components could use the same memorized values? Still not sure. WDYT?

Reply:
I think it cannot hurt to optimize this. `re-resizable` also memoizes the function.

Other components likely have different snap targets though, so it's mostly useful on a per-component basis.

And yes, extracting makes sense. Did that now.